apps/nextjs/src/components/BinIcon.tsx

```diff
@@ -0,0 +1,18 @@
+const BinIcon = () => {
+  return (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M18.4167 6.80769H15.5V6.23077C15.5 5.77174 15.3156 5.33151 14.9874 5.00693C14.6592 4.68235 14.2141 4.5 13.75 4.5H10.25C9.78587 4.5 9.34075 4.68235 9.01256 5.00693C8.68437 5.33151 8.5 5.77174 8.5 6.23077V6.80769H5.58333C5.42862 6.80769 5.28025 6.86848 5.17085 6.97667C5.06146 7.08486 5 7.23161 5 7.38462C5 7.53763 5.06146 7.68437 5.17085 7.79256C5.28025 7.90076 5.42862 7.96154 5.58333 7.96154H6.16667V18.3462C6.16667 18.6522 6.28958 18.9457 6.50838 19.162C6.72717 19.3784 7.02391 19.5 7.33333 19.5H16.6667C16.9761 19.5 17.2728 19.3784 17.4916 19.162C17.7104 18.9457 17.8333 18.6522 17.8333 18.3462V7.96154H18.4167C18.5714 7.96154 18.7198 7.90076 18.8291 7.79256C18.9385 7.68437 19 7.53763 19 7.38462C19 7.23161 18.9385 7.08486 18.8291 6.97667C18.7198 6.86848 18.5714 6.80769 18.4167 6.80769ZM9.66667 6.23077C9.66667 6.07776 9.72813 5.93102 9.83752 5.82282C9.94692 5.71463 10.0953 5.65385 10.25 5.65385H13.75C13.9047 5.65385 14.0531 5.71463 14.1625 5.82282C14.2719 5.93102 14.3333 6.07776 14.3333 6.23077V6.80769H9.66667V6.23077ZM16.6667 18.3462H7.33333V7.96154H16.6667V18.3462ZM10.8333 10.8462V15.4615C10.8333 15.6145 10.7719 15.7613 10.6625 15.8695C10.5531 15.9777 10.4047 16.0385 10.25 16.0385C10.0953 16.0385 9.94692 15.9777 9.83752 15.8695C9.72813 15.7613 9.66667 15.6145 9.66667 15.4615V10.8462C9.66667 10.6931 9.72813 10.5464 9.83752 10.4382C9.94692 10.33 10.0953 10.2692 10.25 10.2692C10.4047 10.2692 10.5531 10.33 10.6625 10.4382C10.7719 10.5464 10.8333 10.6931 10.8333 10.8462ZM14.3333 10.8462V15.4615C14.3333 15.6145 14.2719 15.7613 14.1625 15.8695C14.0531 15.9777 13.9047 16.0385 13.75 16.0385C13.5953 16.0385 13.4469 15.9777 13.3375 15.8695C13.2281 15.7613 13.1667 15.6145 13.1667 15.4615V10.8462C13.1667 10.6931 13.2281 10.5464 13.3375 10.4382C13.4469 10.33 13.5953 10.2692 13.75 10.2692C13.9047 10.2692 14.0531 10.33 14.1625 10.4382C14.2719 10.5464 14.3333 10.6931 14.3333 10.8462Z"
+        fill="#222222"
```

Comment:
If you use fill currentColor then it will use the current text colour and make this more reusable